SOURCES/src/cdlink/CDLINK.ASM

```diff
@@ -13,14 +13,14 @@
 ;	    Edit out reporting of "." and ".." in a root directory.  ;
 ;	    Adjust for "." in a path name.			     ;
 ;   9/01/92 Do stack switch in Executor.  Correct "copyname" proc    ;
-;	    to truncate target filename.ext to 8.3 characters.	     ;
+;	    to truncate target file name.ext to 8.3 characters.	     ;

```

Comment:
In this instance the original was correct as `filename.ext` is intended to be in '8 characters' + '.' + 3 characters format.